Minio/MinioClient.cs

```diff
@@ -87,13 +87,18 @@ public MinioClient()
         Region = "";
         SessionToken = "";
         Provider = null;
-        HTTPClient = new HttpClient();
+        var clientHandler = new HttpClientHandler();
+        clientHandler.ServerCertificateCustomValidationCallback =
+            (sender, cert, chain, sslPolicyErrors) => { return true; };
+        clientHandler.UseProxy = false;
```

Comment:
You can't do this either - we should honor local system proxies.

Reply:
OK.
I'll remove it.